workbench-core/authentication/src/plugins/cognitoAuthenticationPlugin.test.ts

```diff
@@ -711,7 +713,9 @@ describe('CognitoAuthenticationPlugin tests', () => {
     });
 
     it('should throw PluginConfigurationError when the service doesnt have correct permissions', async () => {
-      cognitoMock.on(DescribeUserPoolClientCommand).rejects(new ResourceNotFoundException({ $metadata: {} }));
+      cognitoMock
+        .on(DescribeUserPoolClientCommand)
+        .rejects(new ResourceNotFoundException({ $metadata: {}, message: '' }));
```

Comment:
Hmm, why did the tests change? Does the new version of AWS sdk add the message field?

Reply:
Yeah, it changed message from being optional to being required. I looked through the change logs and didnt see any other behavior changes reported